utils/is.ts

```diff
@@ -57,3 +58,9 @@ export function isUndefinedOrNull(x: any): x is undefined | null {
 export function isNotUndefinedOrNull(x: any) {
   return !isUndefinedOrNull(x)
 }
+
+export function isArguments(...x: any) {
+  return Object.prototype.toString.call(arguments) === '[object Arguments]' ?
```

Comment:
make it more compact. use `is` function for this type checking. remove semicolons.